Polished up Review List and added helpful buttons

JakeSifo opened the pull request.

JakeSifo committed:
Polished up Review List and added helpful buttons

JakeSifo committed:
Polished up Review List and added helpful buttons

JakeSifo committed:
Merge branch 'deleteMe' into ratingsReviews

JakeSifo committed:
Made the buttons glow

JakeSifo committed:
style that works

JakeSifo committed:
Merge branch 'main' into ratingsReviews

JakeSifo committed:
Touch ups

JakeSifo committed:
Progress1

jess-tanonaka reviewed (changes requested):
Looks good overall, but some minor things to clean up:

- [x] **App.jsx:** Can you remove the old appStyles (lines 11 and 12)?
- [x] **R&R index.jsx:** In line 28, you have "w-full" and "max-w-full" in your className. I think the second one might be redundant if you're going to set width to full. Can you check if it is, and remove it if so?
- [x] **ReviewList.jsx:** There's a big ol' comment from lines 11-79. Do you still need that there, or can we put her out to pasture?
- [x] **ReviewListTile.jsx:** At the end of line 41, you have "-mr-5". Does that first "-" do anything, or should it be deleted?
- [x] **ReviewListTile.jsx:** Line 76 has both "mb-2" and "mt-2". These can be condensed into "my-2" if you'd like. It communicates to others that these margins should be the same size and there's no debate on it.

These are mostly trivial so feel free to disregard any of them that you don't see fit to change.

JakeSifo committed:
Got infinite scrolling feature almost done.

JakeSifo committed:
INFINITE SCROLLING (API call a batch then load by 10, call API when at bottom)

JakeSifo committed:
Added infite scroll, fixed getReviews to fit all reviews on 1 page.
Fixed keys for carousel, and all other warning/errors, sorry but too many errors in Overview.
Made changes per Jessica's review.

Blagh
